transitions/multi_form_alerts.js

```diff
@@ -125,33 +132,33 @@ const getPhonesWithDuplicates = (recipients, countedReports) => {
   return getPhonesOneRecipient(recipients, countedReports);
 };
 
-const validateConfig = (alert) => {
```

Comment:
OT: what is up with this diff, github renders it really oddly? Was it tabs before and now it's spaces or something?